### ISSUE TITLE
fails to convert Keras models to ONNX when the model contains multiple `BatchNormalization` layers followed by a custom layer that performs element-wise multiplication in `channels_first` data format

### ISSUE THREAD

**Issue by Inquisitive-ME** [onnx/tensorflow-onnx#2415]
**Describe the bug**
tf2onnx fails to convert Keras models to ONNX when the model contains multiple `BatchNormalization` layers followed by a custom layer that performs element-wise multiplication in `channels_first` data format.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 18.04*): Linux Ubuntu 22.04
- TensorFlow Version: 2.20.0
- Python version: 3.12.9
- ONNX version (if applicable, e.g. 1.11*): 1.16.1
- ONNXRuntime version (if applicable, e.g. 1.11*):

**To Reproduce**
Run the script below it will fail with:
"Process finished with exit code 134 (interrupted by signal 6:SIGABRT)"
[tf2onnx_bug_report.py](https://github.com/user-attachments/files/22895827/tf2onnx_bug_report.py)

**Additional context**
Issue is from trying to use Mobilenetv4Hybrid models in Channels First

**Comment by jaytiwarihub** [comment]
this issue resolved ?

**Comment by jaytiwarihub** [comment]
Root cause seems related to implicit broadcasting in NCHW.

When replacing:
    x * scale

with:
    scale = tf.broadcast_to(scale, tf.shape(x))
    x = tf.multiply(x, scale)

conversion succeeds for both NHWC and NCHW.

This suggests tf2onnx has issues handling implicit broadcasting
for element-wise ops in channels_first format.
